### PR TITLE
[#3] Fix shim script to launch correct version

### PR DESCRIPTION
**Pull request by raxod502**
Before:

```
% ASDF_BUNDLER_VERSION=1.17.3 asdf exec bundle version
Bundler version 2.1.2 (2021-05-12 commit unknown)
% ASDF_BUNDLER_VERSION=2.2.24 asdf exec bundle version
Bundler version 2.1.2 (2021-05-12 commit unknown)
```

After:

```
% ASDF_BUNDLER_VERSION=1.17.3 asdf exec bundle version
Bundler version 1.17.3 (2018-12-27 commit d7089abb6)
% ASDF_BUNDLER_VERSION=2.2.24 asdf exec bundle version
Bundler version 2.2.24 (2021-07-15 commit d78b1ee235)
```

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -15,3 +15,11 @@ if [[ -z "${ASDF_DOWNLOAD_PATH:-}" ]]; then
 else
   cp -R "${ASDF_DOWNLOAD_PATH}"/* "${ASDF_INSTALL_PATH}"
 fi
+
+ls "${ASDF_INSTALL_PATH}/exe" | while read file; do
+    cat "${ASDF_INSTALL_PATH}/exe/${file}" | \
+        sed -E "s|#!/usr/bin/env ruby|#!/usr/bin/env -S RUBYLIB=${ASDF_INSTALL_PATH}/lib ruby|" > \
```
Comment (jonathanmorley) [inline bin/install]: Sed in place instead?

**Comment by jonathanmorley** [comment]
Can we add some regression tests? (even just the examples you listed above)

File: bin/install (hunk 1 above)
Comment (raxod502) [reply bin/install]: Done.

**Comment by raxod502** [comment]
Do you happen to know why the existing test suite didn't find this issue already? I see that you are using [`asdf-vm/actions/plugin-test`](https://github.com/jonathanmorley/asdf-bundler/blob/c42ff91be6fdd3f7c69ad3862c4c8d0ab291961d/.github/workflows/asdf_ci.yml#L29), but perhaps it just runs the command and does not check that the version is correct?

**Comment by dvicory** [comment]
I'm having this issue as well - the default bundler with ruby ends up getting used. Using this branch fixes it. What would it take to move forward on this?

**Comment by raxod502** [comment]
Ah, CI is not passing yet! https://github.com/asdf-vm/asdf/pull/1078 needs to be merged upstream :)

**Pull request merged**